### DISCUSSION TITLE
Like button and creating playlists

### DISCUSSION THREAD

**Discussion by snppls** [leinelissen/jellyfin-audio-player#206] [General]
I've started using this app and it seems great. The only thing I can't work out is how to "like' a song and how to create a playlist? Are these options supported? Will they be in the future?

**Comment by leinelissen** [comment]
Hi @snppls, thanks for your input. Liking songs and creating playlists is currently not supported. At this moment, you'll have to create playlists in the Jellyfin interface, after which they will become available in Fintunes. I'd love to take a look at this, but I'm short on time. I'd love to give pointers to someone willing to build this in themselves.
